db/migrate/20240229125141_change_column_address_in_properties.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeColumnAddressInProperties < ActiveRecord::Migration[7.0]
+  def up
+    change_column(:properties, :address, :jsonb)
+  end
+
+  def down
+    change_column(:properties, :address, :json)
```

Comment:
Why isn't this `:jsonb`?

Reply:
@hiboabd I started with JSON and changed to JSONB just for the sake of consistency 😄